Templates/set_theme_file.R

```diff
@@ -1,6 +1,6 @@
 # This file sets the theme for all the figures
 
-theme_set(theme_minimal(base_size = 12))
+figure_settings <- theme_set(theme_minimal(base_size = 12))
```

Comment:
you shouldn't assign theme_set to anything. It should just work if you run theme_set(my_theme) - but might be better to move theme_set to the RMD file

Reply:
So, do you mean to not have a separate file, define everyting in the Rmd file?

Reply:
I want the definitions in a separate file, so they can be updated globabally, but am not sure if it is better to apply these in the Rmd or the separate file. I sort of like them in the Rmd, so it is clear from the source that we have altered the defaults, but the pragmatic solution is to do it in the separate file - code only needs writing once etc. Can add scale_linetype_auds at some later stage without updating all the Rmd files etc

Reply:
If it works in the separate file, lets go with that. So

```
theme_biostats <- theme_bw(base_size = 14) + ...
theme_set(theme_biostats)

options(ggplot2.continuous.colour="viridis")
```